akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -1,3 +1,6 @@
+---
+project.description: Akka remoting.
+---
```

Comment:
Also not quite a landing page, we'd want people to go to the cluster side and only ever use remoting if they really know what they are doing (tm). 

So perhaps "Details about the underlying remoting module for Akka Cluster."